spring-boot-autoconfigure/src/main/java/org/axonframework/boot/AxonAutoConfiguration.java

```diff
@@ -70,11 +70,8 @@
 @EnableConfigurationProperties(EventProcessorProperties.class)
 public class AxonAutoConfiguration {
 
-    private final EventProcessorProperties eventProcessorProperties;
-
-    public AxonAutoConfiguration(EventProcessorProperties eventProcessorProperties) {
-        this.eventProcessorProperties = eventProcessorProperties;
-    }
+    @Autowired
```

Comment:
Sorry, that was a mistake. The annotation is `@Autowired`. Putting that on the constructor should be enough.